deploy/docker-compose/README.md

```diff
@@ -0,0 +1,261 @@
+# MONAI Deploy Lite
+
+MONAI Deploy Lite provides a simple & lightweight solution for running MONAI Applications (MAPs) on a minimal MONAI Deploy platform.
+
+It is meant to bridge the big gap between the local development of MAPs in a laptop using the MONAI Deploy App SDK and a production-ready deployment. 
+
+Users of MONAI Deploy Lite will be able to run their MAPs, connected to a test PACS, or their own test/research PACS, for further validation, confidently taking steps towards production.   
+
+Reusing the same essential core services for DICOM I/O and AI workflow orchestration provides the same functionality and consistent experience independently of where and how the applications are run, with minimal changes for the end user.
+
+
+## Prerequisites
```

Comment:
The OS is not specified here, explicitly or reference provided.
Since the couple MAPs are included in the example workflow, and they do required GPU support, it is also important to specify GPU and Nvidia Container Toolkit and/or nvidia-docker 2.

Reply:
Yes please specify which OS this works on, and specifically if it works on Mac?

Reply:
Tested on WSLv2 & updated readme.